src/main/scala/firrtl/FileUtils.scala

```diff
@@ -46,9 +46,9 @@ object FileUtils {
   /**
     * recursively delete all directories in a relative path
     * DO NOT DELETE absolute paths
-    * @todo deprecate java.io.File
     * @param file: a directory hierarchy to delete
     */
+  @deprecated("Use deleteDirectoryHierarchy(directoryPathName: String)", "FIRRTL 1.4.3")
```

Comment:
`@deprecated("Use os-lib instead, this function will be removed in FIRRTL 1.6", "FIRRTL 1.5")`

Reply:
Fixed in 58c6479